### ISSUE TITLE
When is the new release on bootstrap 5?

### ISSUE THREAD

**Issue by RedCatGirl** [kristopolous/BOOTSTRA.386#123]
Hi @kristopolous 😘
When is the new release **386** on **bootstrap 5**?

**Comment by kristopolous** [comment]
Just forked it this afternoon. Someone sent me money for a floppy. I kinda feel obligated to work for those $20

**Comment by kristopolous** [comment]
I should probably qualify a release as opposed to each one being some kind of weird art project.

**Comment by kristopolous** [comment]
luckily unlike in every other version, 5 looks pretty similar to 4 in architecture

**Comment by kristopolous** [comment]
I've been actively working on it. I also made a cute video and updated the website because hey, this isn't supposed to be serious

**Comment by EvanCarroll** [comment]
@kristopolous dude, stop telling people it isn't supposed to be serious. You're making the world Borland Blue and doing God's work.

**Comment by kristopolous** [comment]
This is the new demo page I'm going to make to replace the current one

http://9ol.es/tmp/demo.html

it's not done yet ... but it will load each version accordingly.

**Comment by fuxx** [comment]
How do you progress @kristopolous ? No one simply starts a Borland Theme without finishing it :smile:

**Comment by EvanCarroll** [comment]
I agree @kristopolous FINISH VERSION FIVE DUDE. We're all rooting for you. Get it done. Quit your job. This is for the glory. The demo is broken. All the assets are 404.

**The world lacks blue, and it needs YOU.**

https://kristopolous.github.io/BOOTSTRA.386/v5.3.1/

**Comment by kristopolous** [comment]
Oh hi. Alright

**Comment by fuxx** [comment]
👀 

/me hands over some ☕ and 🍕 .

**Comment by BenMcLean** [comment]
I should still like this to be finished please.

**Comment by kristopolous** [comment]
yeah i know ... it's not a theme, it's a proper fork with like 250-400 hours of work.

**Comment by BenMcLean** [comment]
I just started work on a new hobbyist project using version 4 only because version 5 hasn't been made ready. If the original dev doesn't want to continue or hasn't the time necessary for this, maybe some new contributors and/or a fork might assist? I do not have the web experience necessary to be of much help yet myself but maybe sometime if there's no one else.

**Comment by kristopolous** [comment]
alright fine. I should at do some documentation
